files/install_config_agent_yum.sh

```diff
@@ -38,7 +38,7 @@ retry subscription-manager attach --pool=$openstackPoolId
 retry subscription-manager attach --pool=$openshiftPoolId
 retry subscription-manager repos --disable=*
 retry subscription-manager repos --enable=rhel-7-server-rpms --enable=rhel-7-server-extras-rpms --enable=rhel-7-fast-datapath-rpms --enable=rhel-7-server-openstack-9-rpms --enable=rhel-7-server-openstack-9-director-rpms --enable=rhel-7-server-rh-common-rpms
-retry yum -y install os-collect-config python-zaqarclient os-refresh-config os-apply-config openstack-heat-templates python-oslo-log python-psutil ansible
+retry yum -y install os-collect-config python-zaqarclient os-refresh-config os-apply-config openstack-heat-templates python-oslo-log python-psutil ansible-2.4.0.0-5.el7
```

Comment:
might be easier to read if it was split over several lines (e.g. one line per package)